.gitignore

```diff
@@ -6,6 +6,8 @@ docs/_build/
 Minecraft_Overviewer.egg-info
 terrain.png
 cachedir*
+debug_config.py
```

Comment:
please don't, you can keep those in your `.git/info/exclude` as they are specific to you

Reply:
Reverted 👍